pages/Arley-AV/index.html

```diff
@@ -0,0 +1,388 @@
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                    </div>
+                  </li>
+                  <li class="location">
+                    <span class="location__name"
+                      >Mella Santiago de Cuba img</span
+                    >
+                    <div class="location__stars-rating">
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                      <span class="stars-rating__star"
+                        ><img
+                          src="https://upload.wikimedia.org/wikipedia/commons/2/29/Gold_Star.svg"
+                          alt="Icon - Star"
+                      /></span>
+                    </div>
+                  </li>
+                </ul>
+                <button class="card__button">
+                  More
+                  <img
+                    class="button__icon"
+                    src="images/icons/a-left.svg"
+                    alt="Arrow Icon"
+                  />
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="slider__controls">
+          <div class="control">
+            <img
+              src="images/icons/arrow-slider-r.svg"
+              alt="Icon - Arrow Slider Right"
+            />
+          </div>
+          <div class="control control--left">
+            <img
+              src="images/icons/arrow-slider-l.svg"
+              alt="Icon - Arrow Slider Left"
+            />
+          </div>
+        </div>
+        <div class="slider__indicators">
+          <a href="#" class="indicator"></a>
```

Comment:
`indicator` looks more like an element than a block, what do you think could be the block that contains it?

Reply:
Usually, an `a` tag is used to send you to another section or page... do you think that a `button` is a tag that represents better this functionality?